src/deploy-actions.js

```diff
@@ -72,13 +72,14 @@ async function deployActions (config, deployConfig = {}, logFunc) {
     pkg.version = config.app.version
     for (const [name, action] of Object.entries(pkg.actions)) {
       // change path to built action
-      action.function = path.join(relDist, pkgName + '-' + name + '.zip')
+      const zipFileName = (modifiedConfig.ow.package === pkgName ? name : pkgName + '-' + name) + '.zip'
```

Comment:
This logic to create action name based on default package repeats for action build as well, maybe we can move this to utils and use it for build and deploy scripts as required.